L1Trigger/DTTriggerPhase2/src/HoughGrouping.cc

```diff
@@ -683,18 +679,12 @@ ProtoCand HoughGrouping::associateHits(const DTChamber* thechamb, double m, doub
           GlobalPoint tmpGlobal_r = thechamb->superLayer(sl)->layer(l)->toGlobal(tmpLocal_r);
           LocalPoint tmpLocalCh_r = thechamb->toLocal(tmpGlobal_r);
 
-          distleft = std::abs(thepoint.x() - tmpLocalCh_l.x());
-          distright = std::abs(thepoint.x() - tmpLocalCh_r.x());
-
           // Filling info
           returnPC.nLayersWithHits_++;
           returnPC.isThereNeighBourHitInLayer_[abslay] = true;
 
-          returnPC.xDistToPattern_[abslay] = abs(tmpx - (tmpLocalCh.x() - 1.05));
```

Comment:
Don't we need these three returnPC(s) at all? Were it there on purpose?

Reply:
They were overwritten two lines below...

Reply:
Ah, OK. Thanks. 
There is `else` of `if (isDistRight)` which did not show up on my screen.